flask_testing/utils.py

```diff
@@ -295,8 +295,9 @@ def assertRedirects(self, response, location, message=None):
             server_name = self.app.config.get('SERVER_NAME') or 'localhost'
             expected_location = urljoin("http://%s" % server_name, location)
 
-        not_redirect = "HTTP Status 301 or 302 expected but got %d" % response.status_code
-        self.assertTrue(response.status_code in (301, 302), message or not_redirect)
+        valid_status_codes = (301, 302, 303, 305, 307)
+        not_redirect = "HTTP Status 301, 302, 303, 305 or 307 expected but got %d" % response.status_code
```

Comment:
This line seems easy to forget to change if for whatever reason the set of acceptable codes expands or contracts. I would change this to either "Expected HTTP status to be valid redirect..." or join the codes together via commas